etc/sca/windows/cis_win_server_memberL1.rcl.yml

```diff
@@ -0,0 +1,2106 @@
+# Security Configuration assessment
+# CIS Checks for Windows 2012 R2 Member Server L1
+# Copyright (C) 2015-2019, Wazuh Inc.
+#
+# This program is a free software; you can redistribute it
+# and/or modify it under the terms of the GNU General Public
+# License (version 2) as published by the FSF - Free Software
+# Foundation
+#
+# Based on:
+# Center for Internet Security Benchmark for Microsoft Windows Server 2012 R2 v2.3.0 - 03-30-2018
+# Center for Internet Security Benchmark for Microsoft Windows Server 2012 R2 v2.1.0 - 11-02-2015
+# Center for Internet Security Benchmark for Microsoft Windows Server 2012 R2 v2.0.0 - 09-30-2015
+
+policy:
+  id: "cis_win_server_memberL1"
```

Comment:
Please, specify which is the target of this policy. As I told you in a preview review, each SCA policy should point to a specific OS.